lib/Doctrine/DBAL/RewindableGenerator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Doctrine\DBAL;
+
+class RewindableGenerator implements \IteratorAggregate
```

Comment:
Left the RewindableGenerator in place in case users wish to use it from user land but can remove if you'd prefer?